packages/components/src/client/components/Tag.ts

```diff
@@ -0,0 +1,78 @@
+// Developer beware: this file is symlinked between components/ and theme/
+
+import { defineComponent, h,  } from "vue";
+import { useRoute, useRouter } from "vue-router";
+
+import type { VNode } from "vue";
+
+import "../styles/tag.scss";
+
+// TODO: Right now the magic number 9 is baked in
+// That's because access to the theme config is
+// tricky in non-theme components.
+//
+function colorIndex(name: string) {
```

Comment:
arrow function with types declaring is preffered. Just presonal taste. Arror functions can avoid hositing.